File: OldTrello/src/app/Entities/Task.ts
 export class Task{

  id: number | null;
  taskName: string;
  taskDescription: string;
  columnId: number | null;
  lastUpdated: Date;


  constructor(){
    this.id = null;
    this.taskName = "";
    this.taskDescription = "";
    this.columnId =  null;
    this.lastUpdated = new Date();


  }



}

File: OldBackend/vorstu/src/main/java/dev/vorstu/entity/Task.java
package dev.vorstu.entity;


import com.fasterxml.jackson.annotation.JsonBackReference;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Date;

@Entity(name = "task")
@Table(name = "tasks")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Task {

    @Id
    //@GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String taskName;

    private String taskDescription;

    @JsonBackReference
    @ManyToOne
    @JoinColumn(name="column_id", nullable=false)
    private Column column;


    private Long taskPosition;

    private Date lastUpdated;


    // Files
}
